BiblioPixelAnimations/strip/LarsonScanners.py

```diff
@@ -19,6 +19,8 @@ def __init__(self, led, color, tail=2, start=0, end=-1):
 
         self._direction = -1
         self._last = 0
+        if self._tail == 0:
+            self._tail = 1
```

Comment:
Ooh, sloppy on our end!  :-/ 

These things happen.

Thanks for catching this!

Your fix looks as if it's fine...

but my theory is if there's one issue, there's another hiding too :-) so let's page @adammhaile and see what he says about the intent - particularly that weird line 16...